### PR TITLE
fixed unnamed function

### PR DESCRIPTION
**Pull request by marcwelz**
Fixes #

# Merge Checklist:
- [x] Code selfreview has been performed ([Best Practices](https://github.com/axa-ch-webhub-cloud/pattern-library/blob/develop/CONTRIBUTION.md#best-practices)).
- [x] Tests are sufficient.
- [x] Modifications still work in IE.
- [x] Documentation is up to date.
- [x] Changelog is up to date.
- [x] Typescript typings for properties are up to date.
- [x] Designers approved changes or no approval needed.
- [x] Dependencies to other components still work.

### CODE REVIEW

File: src/components/30-organisms/modal/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## 2.0.1
```
Comment (01231) [inline src/components/30-organisms/modal/CHANGELOG.md]: @axa-ch/modal has currently version 
2.0.4

**Review by 01231 — CHANGES_REQUESTED** [review]
LGTM, but you should change the version of the modal in the CHANGELOG.md to 2.0.5

File: src/components/30-organisms/modal/CHANGELOG.md (hunk 1 above)
Comment (marcwelz) [reply src/components/30-organisms/modal/CHANGELOG.md]: I can't see a 2.0.4 version somewhere, even in the pattern-library its 2.0.0
[https://patterns.axa.ch/?path=/story/components-modal--modal](url)

Comment (01231) [reply src/components/30-organisms/modal/CHANGELOG.md]: You can see it on npm

**Review by 01231 — APPROVED** [review]

**Pull request merged**